beacon_node/network/src/sync/block_lookups/tests.rs

```diff
@@ -29,15 +31,44 @@ type T = Witness<ManualSlotClock, CachingEth1Backend<E>, E, MemoryStore<E>, Memo
 struct TestRig {
     beacon_processor_rx: mpsc::Receiver<WorkEvent<E>>,
     network_rx: mpsc::UnboundedReceiver<NetworkMessage<E>>,
+    network_rx_queue: Vec<NetworkMessage<E>>,
     rng: XorShiftRng,
     harness: BeaconChainHarness<T>,
+    sync_manager: SyncManager<T>,
+    beacon_processor: Arc<NetworkBeaconProcessor<T>>,
```

Comment:
Could we have some of the field documentation back as I think it would help people writing the tests understand what each of these fields are for, without reading through the implementation.

https://github.com/sigp/lighthouse/pull/5530/files#diff-13439a1c05bab56ff7fb8b91e2d4f569076456e8a909000e1d9d3b5df8afb1b0R59-R81

Reply:
Added!